assets/css/case-studies.scss

```
@@ -1187,6 +1186,14 @@ $cs-mint-faint: rgba(42, 217, 194, 0.06);
       color: $cs-mint;
       padding-top: 0.35rem;
     }
+    // default inline attribution (name beside role): drop the stacked-layout
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)